PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1

```diff
@@ -19,6 +19,12 @@ function Export-AADProvider {
         # - 2.17 first part
         $AllPolicies = Get-MgIdentityConditionalAccessPolicy -ErrorAction Stop | ConvertTo-Json -Depth 10
 
+        # Set to empty array if returned value is null
+        if ($null -eq $AllPolicies) {
+            # Assign an empty array
+            $AllPolicies = ConvertTo-Json @()
+        }
+
```

Comment:
Thanks for the change.  We're actually going to make this change across the board (not just when the policies are null); see for example the new version of the code that just landed today in ExportEXOProvider (we just couldn't squeeze in AAD unfortunately).

So if its ok by you, we would prefer to delay this PR until we make that change.  We're also planning to move our internal development into this public repo soon rather than dropping bundles of code, so it shouldn't be too long until we land a version of this at HEAD.

Reply:
Yeah no worries. Just thought it would be rude not to submit a PR when the issue was so simple. Feel free to abandon if you prefer.